core/storage/exploration/gae_models.py

```diff
@@ -201,6 +201,19 @@ class ExplorationRightsModel(base_models.VersionedModel):
     )
 
     def save(self, committer_id, commit_message, commit_cmds):
+        """Calls super class method 'commit' to save the version of the
```

Comment:
Caller doesn't need to know implementation details. Just: "Saves a new version of the exploration, updating the Exploration datastore model."

(In general, when you say "the X", it should be clear what that refers to. Here, you say "the version" but there is no reference to version anywhere in the arguments or earlier in the docstring.)

Reply:
Done, thanks for the details.